sdk/storage/azdatalake/internal/generated/autorest.md

```diff
@@ -146,7 +146,7 @@ directive:
 
 ``` yaml
 directive:
-- from: zz_models.go
+- from: zz_options.go
```

Comment:
The below transform should be applied to zz_models.go

Reply:
This is corrected in the next commits.